Sources/ShowAttachments.php

```diff
@@ -291,7 +291,6 @@ function showAttachment()
 	// Does this have a mime type?
 	elseif (!empty($file['mime_type']) && (isset($_REQUEST['image']) || !in_array($file['fileext'], array('jpg', 'gif', 'jpeg', 'x-ms-bmp', 'png', 'psd', 'tiff', 'iff'))))
 		header('content-type: ' . strtr($file['mime_type'], array('image/bmp' => 'image/x-ms-bmp')));
-
```

Comment:
For better readability please leve the empty line separating the conditionals.

Reply:
Done